File: output/llama_4_maverick/avatar/algo_based_translation/java/python/atcoder_ABC149_B.py

def main():
    A, B, K = map(int, input().split())
    x = A
    y = B
    x = A - K
    if x < 0:
        y = B + x
        x = 0
        if y < 0:
            y = 0
    print(x, y)

if __name__ == "__main__":
    main()

File: output/deepseek_chat/avatar/algo_based_translation/python/java/atcoder_ABC149_B.java
import java.util.Scanner;
import java.util.Arrays;

public class atcoder_ABC149_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int[] cookies = Arrays.stream(input).mapToInt(Integer::parseInt).toArray();
        int leftOver = cookies[0] - cookies[2];
        int takahashi = Math.max(0, leftOver);
        if (takahashi > 0) {
            System.out.println(takahashi + " " + cookies[1]);
        } else {
            int remaining = Math.max(0, cookies[1] - Math.abs(leftOver));
            System.out.println(takahashi + " " + remaining);
        }
    }
}
